### ISSUE TITLE
Meta Issue

### ISSUE THREAD

**Issue by subiki** [subiki/fremontopen#72]
### Epic

tournaments

### Summary

Break these into separate issues for development:

Overall Feedback
	Font size is too low
	need to check color ratio for accessibility, gray on gray on black is bad. 
	cinderella run feature - who played above their elo rating in a single tournament 
	
Single Tourney View 
	cant srcoll with tournament bracket in desktop mode
	there is no need for the State column
	we do not need a print button
	for Difficulty, instead of "strong" show me a number or value.
		maybe avg elo or % comparison to avg tournament strength
	There's no need for "started" date. it's just the date, same with State in the overview. 
		
Player pages
	bug - the prev and next buttons glitch, remove them they are not needed 
	remove equipment
	keep number of racks won and lost. 
	keep number of tournaments and ranks. 
	add number of races won and lost.
	add a field for biggest single pot won from the player in a tournament 
	
Leaderboard
	unclear what current and best columns are for. 
	add a filter for total number of tournmanents and racks 
	
	
	

### Why

i dont want to cut a bunch of single issues

### Acceptance criteria

- [ ] ...
all issues created

### Estimated effort

None
